zulipterminal/ui_tools/boxes.py

```diff
@@ -486,9 +486,6 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
-    def selectable(self) -> bool:
-        return True
-
```

Comment:
I'm pretty sure this is not 'dead', but rather necessary? It could benefit from a comment; I believe it allows the user to select a message with a click, or otherwise 'receive focus' in the list - right @amanagr ?

Reply:
Woops! I think you are right here. http://urwid.org/reference/widget.html#urwid.Widget.selectable has information what we want. Perhaps a comment above it as you mentioned would help. 
Sorry for the mistake.

Reply:
I think it would be reasonable to amend this commit to add a comment.

Reply:
Yeah, a comment would be nice there!

Reply:
Done. Updated this with the commit :+1: